common/collation/sv.xml

```diff
@@ -37,5 +37,10 @@ For terms of use, see http://www.unicode.org/copyright.html
 				&[before 1]ǀ<å<<<Å<ä<<<Ä<<æ<<<Æ<<ę<<<Ę<ö<<<Ö<<ø<<<Ø<<ő<<<Ő<<œ<<<Œ<<ô<<<Ô
 			]]></cr>
 		</collation>
+		<collation type="reformed">
+			<cr><![CDATA[
+				[import sv-u-co-standard]
+			]]></cr>
+		</collation>
```

Comment:
Why do we need this?
The regular fallback already goes to the default collation tailoring, which is normally "standard". (It explicitly says so in this file on line 14. That's also the default if that line was missing.)
https://www.unicode.org/reports/tr35/tr35-collation.html#Collation_Type_Fallback

Adding this tailoring here will duplicate the data in ICU, making everyone want to cut it back out.

Reply:
@markusicu Ah thanks for pointing that out, I did not realize that. I have removed the restoration of "reformed" in Swedish collation.